ui/src/reducers/workflow.js

```diff
@@ -68,6 +68,77 @@ export default function workflows(state = initialState, action) {
         terminating: false,
         refetch: true
       };
+    case 'REQUESTED_BULK_TERMINATE_WORKFLOW':
```

Comment:
For readability and code maintainability, you may want to condense some of these where applicable to handle multiple cases with a single return.

e.g. 
`case 'REQUESTED_BULK_TERMINATE_WORKFLOW':
case 'REQUESTED_BULK_RESTART_WORKFLOW':
case 'REQUESTED_BULK_RETRY_WORKFLOW':
case 'REQUESTED_BULK_PAUSE_WORKFLOW':
case 'REQUESTED_BULK_RESUME_WORKFLOW':
return {
         ...state,
         bulkProcessInFlight:true,
         bulkProcessSuccess:false
       };
`

Reply:
Refactored in latest commit